packages/chrome/src/elements/Chrome.tsx

```diff
@@ -38,6 +40,27 @@ const Chrome = React.forwardRef<HTMLDivElement, IChromeProps & ThemeProps<Defaul
     const isLight = hue ? isLightMemoized : false;
     const isDark = hue ? !isLightMemoized : false;
     const chromeContextValue = { hue: hue || 'chromeHue', isLight, isDark };
+    const environment = useDocument(theme);
+
+    useEffect(() => {
+      if (!environment) {
+        return undefined;
+      }
+
+      const htmlElement = environment.querySelector('html');
+
+      if (htmlElement && isFluid) {
```

Comment:
"fluid" is the opposite of "fixed". The prop should stay defined as-is. However if `isFluid={true}` then no `position` style is added. Otherwise, `position: fixed` is added to the `<html>` element by default.

Consequently `isFluid` would be added to all styleguidist examples so that the page continues to scroll.

Reply:
Recommend changing the title of this PR to "fix(chrome): add fixed positioning to html element"